scaling_performance/cluster_maximums.adoc

```diff
@@ -146,14 +149,41 @@ application requirements.]
 | 5,000
 | 5,000
 
-| Number of Deployments per Namespace footnoteref:objectpernamespace[]
+| Number of Deployments per Namespace ^[2]^
 | 2,000
 | 2,000
 | 2,000
 | 2,000
 
 |===
 
+[.small]
+--
+1. The Pod count displayed here is the number of test Pods. The actual number of
+Pods depends on the application’s memory, CPU, and storage requirements.
+2. There are a number of control loops in the system that need to iterate over all
+objects in a given namespace as a reaction to some changes in state. Having a
+large number of objects of a given type in a single namespace can make those
+loops expensive and slow down processing given state changes. The maximum
+assumes that the system has enough CPU, memory, and disk to satisfy the
+application requirements.
+3. Each Service port and each Service back-end has a corresponding entry in
+iptables. The number of back-ends of a given service impact the size of the
+endpoints objects, which impacts the size of data that is being sent all over
+the system.
+--
+[[scaling-performance-3-11-route-maximums]]
+=== Route Maximums
+
+In 3.11.53, router tests were completed in a 3-node environment on Amazon Web
+Services (AWS). There were 100 HTTP routes, specifically 100 back-end Nginx pods,
+with `keepalive` set to `100`, and 200 parallel connections per route. The
```

Comment:
you can remove 200 parallel connections per route as we are stating numbers for 1,40 and 200 connections per target below